PyFlyt/core/aviary.py

```diff
@@ -17,10 +17,11 @@ def __init__(
         self,
         start_pos: np.ndarray,
         start_orn: np.ndarray,
+        start_vel: np.ndarray,
         render: bool = False,
         physics_hz: int = 240,
         ctrl_hz: int = 120,
-        drone_model: str = "cf2x",
+        drone_model: str = None,
```

Comment:
str cannot be instantiated as None, use "".

Reply:
Reverted to `"cf2x"`. We could use this to select FW or quadx at the spawn drones section in this file

Reply:
Tried to implement that, but it will require an `np.ndarray` containing strings of `drone_model` names, but file paths are expecting `str`. Some modifications are required if want to do this way